scalability_and_performance/planning-your-environment-according-to-object-limits.adoc

```diff
@@ -0,0 +1,30 @@
+[id='planning-your-environment-according-to-object-limits']
+= Planning your environment according to object limits
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: object limits
+
+toc::[]
+
+
+Consider the following object limits when you plan your {product-title} cluster.
```

Comment:
Before GA, I think you'll need another paragraph or two about the difference in planning between BYO and next-gen clusters. Some of the modules might only apply to BYO.